dashboard/settings.py

```diff
@@ -25,35 +23,37 @@
 PROJECT_ROOT = os.path.abspath(
     os.path.join(os.path.dirname(__file__), ".."),
 )
+with open(os.getenv("ENV_FILE", "/code/config/env.json")) as f:
+    ENV = json.load(f)
 
 LOGOUT_URL = '/accounts/logout'
 
 # Google Analytics ID
-GA_ID = config('GA_ID', default='')
+GA_ID = ENV.get('GA_ID', '')
 
 # Quick-start development settings - unsuitable for production
 # See https://docs.djangoproject.com/en/1.9/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = config('DJANGO_SECRET_KEY')
+SECRET_KEY = ENV.get('DJANGO_SECRET_KEY')
```

Comment:
default value needed?

Reply:
Well it won't start up unless you have one in the environment file! I think Django failing to start because the secret_key isn't defined is okay.